### PR TITLE
 Improve handling of incoming calls

### PR DESCRIPTION
**Pull request by jznadams**
Add a fix to handle missed incoming calls. 
Add test cases for incoming calls.
Fix weird index problem on the group list activity after finishing a call from a test case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,75 @@ When you run the application, you will be given the chance to enter an endpoint
...
+
+    $ "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" \
+    --use-fake-ui-for-media-stream \
+    --use-fake-device-for-media-stream \
+    --allow-file-access-from-files \
+    ./WebTestBot/index.html &
+
+2) Once the file has loaded, append your local username and Respoke application ID to the URL to match what Android Studio will search for as the tests run:
+
+    file:///respoke-android-sdk/WebTestBot/index.html#?un=mymacusername&app_id=my-respoke-app-id
```
Comment (chadxz) [inline README.md]: references android sdk instead of sample

File: WebTestBot/index.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    </head>
+    <body>
+        <div ng-controller="RespokeController">
+
+            <h3 ng-show="client.isConnected()">
+                Connected as {{username}}. Waiting for test users.
+            </h3>
+
+            <div ng-show="!client.isConnected()">
+                Connecting the SDK autotest bot, please wait...
```
Comment (chadxz) [inline WebTestBot/index.html]: s/sdk/sample app

**Comment by chadxz** [comment]
LGTM

File: WebTestBot/index.html (hunk 2 above)
Comment (jznadams) [inline WebTestBot/index.html]: Fixed

File: README.md (hunk 1 above)
Comment (jznadams) [inline README.md]: Fixed

**Pull request merged**